src/include/duckdb/parser/parsed_data/create_type_info.hpp

```diff
@@ -26,13 +26,18 @@ struct CreateTypeInfo : public CreateInfo {
 	string name;
 	//! Logical Type
 	LogicalType type;
+	//! Used by create enum from query
+	std::unique_ptr<SQLStatement> query;
```

Comment:
nit: no `std::`, I believe `common/helper.hpp` has the `use std::unique_ptr` declaration in it